src/main/java/org/fogbowcloud/manager/core/models/orders/ComputeOrder.java

```diff
@@ -19,24 +19,37 @@
 	/** Disk attribute, must be set in GB. */
 	@Column(name = "disk")
 	private int disk;
-	
+
 	@Column(name = "image_name")
 	private String imageName;
 
 	@OneToOne(cascade = CascadeType.ALL)
 	@JoinColumn(name = "user_data_id")
 	private UserData userData;
 
-	public ComputeOrder(){ }
+	public ComputeOrder() {
+	}
 
```

Comment:
do we need this empty constructor? Is it ok to remove these database annotations right now or it is better to remove it latter?

Reply:
Done, [here](https://github.com/fogbow/fogbow-manager-core/pull/13/commits/8720df9422851ec7995ce6eabb3696d670490d58).